### DISCUSSION TITLE
Release 2.6.0

### DISCUSSION THREAD

**Discussion by stelgenhof** [azuyalabs/yasumi#313] [Announcements]
There is a fresh release at last! I really believe in releasing software as frequently as possible, yet I somehow neglected to do so for more than a year. Even though a lot of work was completed during that time, I was just unable to pick the ideal moment to make an official release.

Despite the fact that no new countries are included in this release, there are several upgrades, corrections, and refinements that make Yasumi 2.6 worthwhile to upgrade to.

Below you will find a summary of the most significant changes. For more details, please check the [CHANGELOG](https://github.com/azuyalabs/yasumi/blob/master/CHANGELOG.md) file of this project or navigate to the [v2.6 release](https://www.yasumi.dev/blog/release_v26/) blog post.

### Implemented Enhancements

- Bank holiday for King Charles III’s Coronation in the United Kingdom. [\#305](https://github.com/azuyalabs/yasumi/pull/305) ([Freshleaf Media](https://www.github.com/freshleafmedia))
- Bank holiday for Queen Elizabeth II’s State Funeral on September 19, 2022, for the United Kingdom. [\#287](https://github.com/azuyalabs/yasumi/pull/287) ([Freshleaf Media](https://www.github.com/freshleafmedia))
- National Day of Mourning for Australia. [\#288](https://github.com/azuyalabs/yasumi/pull/288) ([FuzzyWuzzyFraggle](https://www.github.com/FuzzyWuzzyFraggle)).
- In Japan, Marine Day was rescheduled to July 23 as the 2020 Tokyo Olympics took place. The rescheduled Marine Day for
  2021 was included, but not the original rescheduled day for 2020.
- Slovak translations for a couple of popular holidays. [\#298](https://github.com/azuyalabs/yasumi/pull/298) ([Jozef Grencik](https://www.github.com/jozefgrencik))

### Resolved Issues

- Liberation Day for The Netherlands is only an official holiday every 5 years [\#280](https://github.com/azuyalabs/yasumi/pull/280) ([Daan Roet](https://github.com/droet)).
- Pentecost Monday in France was only recognized as an official holiday until 2004. Since 2004, it is considered a
  special holiday, a so called 'working holiday'. Hence, it is therefore classified as an observed holiday in Yasumi
  from 2004 and forward. [\#281](https://github.com/azuyalabs/yasumi/issues/281).
- The holiday of Epiphany (6th of January) was incorrectly categorized as `other` and changed to an official holiday in
  Baden-Württemberg, Bavaria and SaxonyAnhalt. [\#296](https://github.com/azuyalabs/yasumi/issues/296) ([Anna Damm](https://github.com/AnnaDamm)).
- The year 1988 was incorrectly omitted from observing the Emperor's birthday in Japan.
- The tests for Remembrance Day, Malvina's Day and National Sovereignty Day in Argentina were considered for all years;
  however, these have only been celebrated since their establishment.
- Tests for New Year's Day, Spring Bank Holiday, and May Day Holiday in the United Kingdom (England, Wales, Northern
  Ireland, and Scotland), as well as Battle of the Boyne in Northern Ireland, were considered for any calendar year;
  however, these are celebrated only since a particular calendar year.
- In version 2022f of the `tz` db, a correction for 1947 was made for the summertime transition in Denmark to April
  the 6th. Various corrections have been made to accommodate for change.

## New Contributors
* @jwojtyra-aterian made their first contribution in https://github.com/azuyalabs/yasumi/pull/277
* @DRoet made their first contribution in https://github.com/azuyalabs/yasumi/pull/280
* @freshleafmedia made their first contribution in https://github.com/azuyalabs/yasumi/pull/287
* @jozefgrencik made their first contribution in https://github.com/azuyalabs/yasumi/pull/298
* @AnnaDamm made their first contribution in https://github.com/azuyalabs/yasumi/pull/297
* @FuzzyWuzzyFraggle made their first contribution in https://github.com/azuyalabs/yasumi/pull/288
* @SupraSmooth made their first contribution in https://github.com/azuyalabs/yasumi/pull/292
* @tylershuster made their first contribution in https://github.com/azuyalabs/yasumi/pull/282

**Full Changelog**: https://github.com/azuyalabs/yasumi/compare/2.5.0...2.6.0

<hr /><em>This discussion was created from the release <a href='https://github.com/azuyalabs/yasumi/releases/tag/2.6.0'>2.6.0</a>.</em>

**Comment by sylvaincombes** [comment]
That's great ! Thank you for your hard work @stelgenhof and also all contributors :tada:
